src/EventStore.Core/Metrics/ElectionsCounterTracker.cs

```diff
@@ -0,0 +1,36 @@
+﻿using EventStore.Core.Bus;
+using EventStore.Core.Messages;
+
+
+namespace EventStore.Core.Metrics;
+
+public interface IElectionCounterTracker {
+	void IncrementCounter();
+}
```

Comment:
```suggestion
public interface IElectionCounterTracker : IHandle<Messages.ElectionMessage.ElectionsDone> {
}```

ah ok sorry, i meant like this ^